### PR TITLE
Add LabelList.vue

### PR DESCRIPTION
**Pull request by andres-holguin**
image
A basic component that turns labels and school/course objects into labels that can display on the note preview.

### CODE REVIEW

File: src/main/webapp/components/LabelList.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="labels-list">
+    <!-- School Label -->
+    <v-chip
+      class="ma-1"
+      color="orange"
+      dark label
+      :close="canModify"
+      close-icon='mdi-find-replace'
```
Comment (Aradhya-B) [inline src/main/webapp/components/LabelList.vue]: nit: I think it's more readable to close tags on the same line as the last line in the tag. Same goes for the tags below.
```suggestion
    close-icon='mdi-find-replace'>
```

**Review by Aradhya-B — APPROVED** [review]
Great work Andres!

File: src/main/webapp/components/LabelList.vue (hunk 1 above)
Comment (keelywan) [reply src/main/webapp/components/LabelList.vue]: +1

**Review by keelywan — APPROVED** [review]
Looks good!

File: src/main/webapp/components/LabelList.vue (hunk 1 above)
Comment (andres-holguin) [reply src/main/webapp/components/LabelList.vue]: Sounds good. I was following the [examples on Vuetify's website](https://codepen.io/pen/?&editable=true&editors=101=https%3A%2F%2Fnext.vuetifyjs.com%2Fen%2Fcomponents%2Falerts%2F) but I think I agree. This also seems to align with [Google's HTML guidelines](https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules).

I'll make those changes.

Comment (cftorres) [reply src/main/webapp/components/LabelList.vue]: LGTM with style changes :)

**Review by cftorres — APPROVED** [review]

**Review by nickKelman — APPROVED** [review]
Nice work

**Pull request merged**